fission/main.go

```diff
@@ -277,13 +277,13 @@ func main() {
 	// canary configs
 	canaryConfigNameFlag := cli.StringFlag{Name: "name", Usage: "Name for the canary config"}
 	triggerNameFlag := cli.StringFlag{Name: "httptrigger", Usage: "Http trigger that this config references"}
-	funcNFlag := cli.StringFlag{Name: "funcN", Usage: "New version of the function"}
-	funcNminus1Flag := cli.StringFlag{Name: "funcN-1", Usage: "Old stable version of the function"}
+	newFunc := cli.StringFlag{Name: "newfunc", Usage: "New version of the function"}
```

Comment:
Can we use the full word "function" in the cli?  (we do this for the triggers too)